.github/workflows/block-build.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       - run: npm run coverage
 
-      - uses: codecov/codecov-action@v1
+      - uses: codecov/codecov-action@v2
```

Comment:
What is the practical effect of this change?

Reply:
nothing really just staying up to latest, from the changelog it says that it just uses their new uploader code, i hoped it would help with the missing coverage files but didn't seem to